scripts/update_simdutf.sh

```
@@ -0,0 +1,10 @@
+#/bin/bash
```

Shellcheck (0.11.0)

[error] Use #!, not just #, for the shebang.

(SC1113)